files/homeassistant/hacs_version/individual_printer_image_a1mini.yaml

```diff
@@ -57,16 +57,18 @@ elements:
         type: state-badge
         style:
           top: 30%
-          left: 40.5%
+          left: 40.8%
           font-size: 0.8em
           color: rgba(0,0,0,0)
+          '--label-badge-red': '#0c0c0c'
```

Comment:
Is this a personal stylistic choice to change the colour of temperature icons to all black? Or is there some functionality/benefit?

Reply:
Just personal syle but I reverted it. Makes sense that HOT = RED 🔥